### ISSUE TITLE
PHP 8.1 - match is formatted into one line if the whole block is short enough - which is weird

### ISSUE THREAD

**Issue by AndreasA** [prettier/plugin-php#1964]
If the used `match` with not a lot of content, it is formatted as one line which is really tricky to read.

**Prettier 2.6.2**

**PHP Plugin 0.18.4**

```json
{
  "printWidth": 80,
  "tabWidth": 4,
  "useTabs": false,
  "singleQuote": true,
  "phpVersion": "8.1",
  "trailingCommaPHP": true,
  "braceStyle": "psr-2",
  "requirePragma": false,
  "insertPragma": false
}
```

**Input:**

```php
<?php declare(strict_types=1);

match ($case) { 
     'foo' => ['bar'], 
     'cd' => [],
     default => [],
};

```

**Output:**

```php
<?php declare(strict_types=1);

match ($case) { 'foo' => ['bar'],  'cd' => [],  default => [] };

```

**Expected behavior:**

To be formatted as it was with input, to put each "case" in a new line.

[**Playground sample**](https://loilo.github.io/prettier-php-playground/#N4IgDgTgpgLjCWUIHkwIPYDsDOIBcok8mMA6vACYwAW+AHAAwA0IMAhgEblW14AsLAK7YoAFU648AMzYAbES2zEA5rKgBFQehhR8MCIKgsw1MADUkSrPhB0AdAEYQLfW3iyVAYXQBbH2wAFAAkAvQMjEA4INgBjKABlGABPNRswbAgAWgAmZxBoAEdBeGgA6OV-fBl5COIRCBgytgq2KrkRAF8WKAp4GHQUNHgsSVA2bBg2mq6QYjBBSbwQAB4AfhMwAAIKKBjZNmgACgmIeBiYAH1ksChsAF4HAEoAbgAdTHf-GBjqTcOAEhi4ygj02wE2702UKhAHIpOh0DDNncAHybADaMI4BxhAF0mBDMNDYTEKEjURj8ZDiTsZIJZDBkWj0VTMB03pgQB0gA)

**Comment by czosel** [comment]
Totally agree, thanks for reporting @AndreasA. The fix for this is potentially quite simple (replace „line“ with „hardline“) at the right place).

**Comment by AndreasA** [comment]
@czosel I think this https://github.com/prettier/plugin-php/pull/1994/files should fix it? Hopefully, it was the correct place. It probably does not fix https://github.com/prettier/plugin-php/issues/1965 though.

**Comment by czosel** [comment]
@AndreasA I also noticed that nested `match` looks a bit strange.  The fix should be relatively simple, see how `indent` is used in the printer. We can do this in a separate PR, though.

Fixed in `v0.18.7` :tada:
